packages/app/src/cli/services/context.ts

```diff
@@ -358,22 +342,10 @@ export interface DeployContextOptions {
  * @param developerPlatformClient - The client to access the platform API
  * @returns The selected org, app and dev store
  */
-export async function ensureDeployContext(options: DeployContextOptions): Promise<DeployContextOutput> {
-  const {reset, force, noRelease} = options
-  let developerPlatformClient = options.developerPlatformClient
-  const enableLinkingPrompt = !options.apiKey && !isCurrentAppSchema(options.app.configuration)
-  const [remoteApp] = await fetchAppAndIdentifiers(options, developerPlatformClient, true, enableLinkingPrompt)
-  developerPlatformClient = remoteApp.developerPlatformClient ?? developerPlatformClient
+export async function ensureDeployContext(options: DeployOptions): Promise<Identifiers> {
+  const {reset, force, noRelease, app, remoteApp, developerPlatformClient} = options
   const activeAppVersion = await developerPlatformClient.activeAppVersion(remoteApp)
 
-  const specifications = await fetchSpecifications({developerPlatformClient, app: remoteApp})
-  const app: AppInterface = await loadApp({
-    specifications,
-    directory: options.app.directory,
-    userProvidedConfigName: getAppConfigurationShorthand(options.app.configuration.path),
-    remoteFlags: remoteApp.flags,
-  })
-
   const org = await fetchOrgFromId(remoteApp.organizationId, developerPlatformClient)
```

Comment:
It's needed for a prompt that needs the org name (the box with all the details about the current app)

Not all commands need the org (most dont), so seemed like something that could be done per-command.
Commands that need the org, is for the same reason: to display the name in some prompt. Do you think we should still fetch the org in `app-context`? 🤔

Reply:
My instinct is yes: it feels like background information that any command _may_ use.

There is a line between what belongs in context and what doesn't (e.g. we could load information on all of an app's modules and releases in context, but we don't), but this feels like something that sits on the context side of the line.

Thinking on this:
- organisation ID is in the TOML, and failure to load the org should be equally as disruptive as failing to load the remote app (by client ID)
- as >1 command needs to know the organisation, we have to make sure there is consistency in how organisations are loaded. including it in context is a good solution for that
- we don't, but if we always showed the same "context box" widget for every app command, it would be expected to see the organisation name in there

Reply:
Ready @shauns